persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/configuration/TableManipulationConfiguration.java

```diff
@@ -101,6 +103,7 @@ public int fetchSize() {
    /**
     * @return the size of batches to process.  Guaranteed to be a power of two.
     */
+   @Deprecated
```

Comment:
Please put this in the javadoc as well stating they should be using the one on the StoreConfiguration.